submissions/ALT+F4 LTN-RLD-Cybersyn Mod Comparison/Requesting a Train.md

```diff
@@ -0,0 +1,101 @@
+
+One planned feature is a [manager GUI](https://mods.factorio.com/mod/cybersyn/discussion/63b1ebbf7c6d7ab3f0ba2285) similar to [LTN Manager](https://mods.factorio.com/mod/LtnManager), which is one of several third-party mods for LTN. This mod gives an overview of the stations in LTN, allowing one to quickly view station inventories, requests, etc. When implemented, Cybersyn's version will have greater feature parity with LTN.
+
+### Rail Logistics Dispatcher - Built-in GUI
+
+The second of the three mods to be released, Rail Logistics Dispatcher, is also the most radically different, with its most prominent feature being an integrated GUI.
+
+{% include compare.html old='RLD=stop.png' new='RLD-stop-ALT.png' caption='The Tititular Rail Logistics Dispatcher. While it resembles a lamp, a look in ALT mode reveals the truth of the matter: It is actually a combinator in disguise!' %}
+
+Much like Cybersyn, Rail Logistics Dispatcher forgoes replacing the vanilla train stop in favor of adding another entity to control it. However, the Dispatcher it adds is smaller, doesn't require power, and uses lamp graphics similar to LTN, making it easier to tell whether the stop is functioning correctly. Taken together, the minimum footprint for the dispatcher and train stop is 2x2 + 1x1, with the latter needing to be 1 tile away from the stop it controls.
+
+{% include gallery.html items='RLD-GUI.png;Rail Logistics Dispatcher's GUI,LTN-Combinator-Modernized.png;LTN Combinator Modernized's GUI' %}
+
+While third party mods like [LTN Combinator Modernized](https://mods.factorio.com/mod/LTN_Combinator_Modernized) have been very helpful to those lacking experience with circuit network, the fact remains that it is an entirely separate mod dedicated to simplfying and improving the user experience. With that in mind, RLD opted to include a GUI right out of the box. (RLD does also offer a 'signals mode' to allow the user to configure the Dispatcher in a manner more similar to LTN if desired.) Similar to Project Cybersyn above, Rail Logistics Dispatcher's author also plans to implement a native [manager UI](https://mods.factorio.com/mod/RailLogisticsDispatcher/discussion/6390043c3a48c93b86a5f22b) to view the whole network in the future.
+
+Unlike both of the previous two mods, Rail Logistics Dispatcher adds no new signals at all. Everything that was previously done through signals such as Thresholds, Train Configurations, Network IDs, is instead done through use of the GUI. RLD also goes all in on the the per-item thresholds. It doesn't have any per-station thresholds through its GUI, nor any global thresholds in its settings. However, it can save certain requests and thresholds to be applied again later. In addition to the individual item units and stacks of items, RLD also has the ability to set requests based on a wagon (fluid or cargo) load.
```

Comment:
```suggestion
Unlike both of the previous two mods, Rail Logistics Dispatcher adds no new signals at all. Everything that was previously done through signals such as Thresholds, Train Configurations, and Network IDs, is now done through the use of the GUI. RLD also goes all-in on the per-item thresholds. It doesn't have any per-station thresholds through its GUI, nor any global thresholds in its settings. However, it can save certain requests and thresholds to be applied again later. In addition to the individual item units and stacks of items, RLD also has the ability to set requests based on a wagon (fluid or cargo) load.
```